aws/ecs/appstore_metrics_etl.tf

```diff
@@ -44,6 +45,7 @@ module "unmasked_appstore_metrics_etl" {
   sg_id                          = var.sg_id
   template_file                  = file("task-definitions/metrics.json")
   event_rule_schedule_expression = var.schedule_expression
+  metric_token_secret_arn        = aws_secretsmanager_secret.metrics_token.id
```

Comment:
Is it possible to do this in a way that doesn't tightly couple the module to what the code is doing?